### DISCUSSION TITLE
Monitor Control doesn't start from time to time

### DISCUSSION THREAD

**Discussion by ronakg** [MonitorControl/MonitorControl#1049] [Q&A]
I'm using Monitor Control 4.1.0 on macOS 12.4 with Samsung LU28R55 4k monitor. From time to time, I notice that monitor control doesn't start at all. Nothing happens when I open the app.

How do I investigate this issue? Are there any logs that I can provide?

**Comment by ronakg** [comment]
Any suggestions?

**Reply by waydabber** [reply]
Hmm. It would be educational to see what is happening in the Console. See this:

#851
